docs/enterprise/kubernetes-guides/eks/cluster.md

```diff
@@ -58,14 +58,19 @@ metadata:
   region: us-west-1
   version: "1.30"
 
-# Node groups that can be used to scale the cluster up and down
+# Availability zones (AZ) (at least two zones must be specified)
+# Note: not all instance types are available in all availability zones
+availabilityZones: ["us-west-1a", "us-west-1c"]
```

Comment:
I'm not sure I follow why we need two here?

Reply:
eksctl command fails if there is less than two: https://docs.aws.amazon.com/eks/latest/userguide/network_reqs.html

> at least two subnets that are in different Availability Zones.

Reply:
If I recall correctly, if they are not specified, eks picks two availability zones by itself, unaware of the availability zones specified in the node groups. If a node group specifies an availability zone which is not picked, eksctl will error.

Reply:
Weird. I guess I never ran into it.